static/assets/icons/arrow-left-circle.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="19" height="19" viewBox="0 0 19 19" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We can use the fontawesome icon here - https://fontawesome.com/v4/icon/arrow-circle-o-left

Reply:
We got it from @terpimost which we liked most from a [few variations](https://chat.zulip.org/#narrow/stream/101-design/topic/narrow.20to.20topic.2Fpms.20when.20composing/near/1316093). The fontawesome one looks a bit different. Do we want that?

Reply:
Hm, could you post screenshots of the two versions (current icon and FontAwesome icon) next to each other?

Reply:
Sure, FontAwesome followed by current:
![image](https://user-images.githubusercontent.com/44665669/158540820-30f889a8-4c90-459f-b91d-927122e44b19.png)
![image](https://user-images.githubusercontent.com/44665669/158540735-bd49de72-c354-4bbf-acee-713e2dcc2440.png)

Reply:
Cool, thanks! I think the current icon looks better, so let's stick with that.